features/claim-mgt/org.wso2.carbon.claim.mgt.server.feature/resources/conf/claim-config.xml

```diff
@@ -756,7 +756,7 @@
 				in your user store must be configured for this. -->
 				<AttributeID>lastPasswordUpdate</AttributeID>
 				<Description>Last Password Update Time</Description>
-				<dataType>date_time</dataType>
+				<dataType>epoch</dataType>
```

Comment:
"The SCIM Schemas response will have the first class data type as DATE_TIME but under the inputFormat object will contains the format as shown below."

Doesn't this contradict with this statement in the original issue?

Reply:
As the SCIM specification defines the date date_time data type, the attribute data types defined as date, epoch can't be applied. Hence the SCIM schemas response will give the data type as DATE_TIME for all those data types, but will have additional detail as format under the inputFormat about the date format.

For the claims, we can have first class data types defined and returned in the claim response as data type as epoch or date.